src/particle.cpp

```diff
@@ -237,6 +246,9 @@ void Particle::event_advance()
     double push_back_distance = speed() * dt;
     this->move_distance(-push_back_distance);
     hit_time_boundary = true;
```

Comment:
Just realized that the time cutoffs will work just fine for running transients, we just need to add some banking logic here if the boundary is less than the time cutoff. yay!